packages/react-ui-kit/src/Form/IndicatorRangeInput.tsx

```diff
@@ -76,15 +78,35 @@ export const IndicatorRangeInput: FC<IndicatorRangeInputProps> = forwardRef<
       return `${((valueNum - minNum) * 100) / (maxNum - minNum)}% 100%`;
     }, [isCustomSlider, valueNum, minNum, maxNum, listLength]);
 
+    const currentSelectedValue = useMemo(() => {
+      if (typeof value !== 'string' && typeof value !== 'number') {
```

Comment:
isStringOrNumber can be used here?

Reply:
Why this check if even needed here? Can't we just explicitly set prop type to be a number? I wonder if `extends TextProps<T>` is needed I feel like we're losing some control here due to very wide prop types. Wdyt?

Reply:
can't we use `valueNum` declared earlier instead of `value`?